lib/meadow/data/collections.ex

```diff
@@ -48,6 +48,18 @@ defmodule Meadow.Data.Collections do
     |> add_representative_image()
   end
 
+  def get_work_count(collection_id) do
+    query =
+      from w in Work,
+        where: w.collection_id == ^collection_id,
+        select: count("*")
+
+    case Repo.one(query) do
+      nil -> {:error, "Error getting work count for collection #{collection_id}"}
+      count -> {:ok, count}
+    end
+  end
```

Comment:
Ecto has a [convenience method](https://hexdocs.pm/ecto/Ecto.Repo.html#c:aggregate/3) for this:
```
from(w in Work, where: w.collection_id == ^collection_id)
|> Repo.aggregate(:count)
```
It always returns either a number or `nil`, so no error checking needed.